virttest/utils_libvirt/libvirt_audit.py

```diff
@@ -0,0 +1,38 @@
+"""
+libvirt audit related utility functions
+"""
+
+from avocado.utils import process
+from avocado.utils import service
+
+from virttest import utils_libvirtd
+
+
+def enable_audit_log(libvirtd_config):
+    """
+    Configure audit log level"
```

Comment:
Do you need to delete the '"' at the end of the line?

Reply:
removed